_azure/deploy.ps1

```diff
@@ -0,0 +1,19 @@
+﻿Set-StrictMode -Version 3
+Import-Module Azure -ErrorAction SilentlyContinue
+
```

Comment:
Would be good to have the script exit if $EnvName or $BranchName are empty. Same with $AzureSub below.